core/desugarEffects.ml

```diff
@@ -548,11 +548,11 @@ let collect_operation_of_type tp
       | Some vid -> RowVarMap.add_raw (-1)
                       (RowVarMap.find_raw vid operations) operations
     in
-    (* print_endline "Collected operations:";
-     * print_endline
-     *   (RowVarMap.show (fun ppr x -> Format.fprintf ppr "%s"
-     *                                   (StringSet.show x)) operations);
-     * flush_all (); *)
+    print_endline "Collected operations:";
```

Comment:
I don't think these should be printed unless `debug` is enabled (and possibly some other option). Maybe just delete them again.

Reply:
Seems like I was viewing an outdated commit.